modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSourceReact.testcase

```diff
@@ -43,15 +43,6 @@ definition {
 
 	}
 
-	@description = "Verify that the user is able to import a XML file"
```

Comment:
[LRQA-74659](https://liferay.atlassian.net/browse/LRQA-74659)  This test is already covered in several other tests